ProfessorAssessor/View Controllers/LoginViewController.m

```diff
@@ -18,19 +23,77 @@ - (IBAction)login:(UIButton *)sender {
     NSString *password = self.password.text;
 
     if ([self validCredentials]) {
-        [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser *user, NSError *error) {
+        [PFUser logInWithUsernameInBackground:username
+                                     password:password
+                                        block:^(PFUser *user, NSError *error) {
             if (error == nil) {
-                UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
-                HomeViewController *viewController = [storyboard instantiateViewControllerWithIdentifier:@"HomeViewController"];
-
-                SceneDelegate *sceneDelegate = (SceneDelegate *)self.view.window.windowScene.delegate;
-
-                [sceneDelegate changeRootViewController:viewController];
+                [self displayHomePage];
             }
         }];
     }
 }
 
+- (IBAction)continueWithFacebook:(UIButton *)sender {
+    FBSDKLoginManager *manager = [FBSDKLoginManager new];
+    FBSDKLoginConfiguration *configuration = [[FBSDKLoginConfiguration alloc] initWithPermissions:@[@"public_profile", @"email"] tracking:FBSDKLoginTrackingEnabled];
```

Comment:
nit: new lines for parameters of long methods. Also, you may be able to omit the tracking param. I think there's a default override for this method that just takes permissions.